src/renderer/components/DeviceAction/index.js

```diff
@@ -123,6 +124,17 @@ const DeviceAction = <R, H, P>({
     }
   }, [dispatch, modelId, preferredDeviceModel]);
 
+  useEffect(() => {
+    if (deviceInfo) {
+      const lastSeenDevice = {
+        modelId: device.modelId,
```

Comment:
is device always set when we have deviceInfo ? Or do device?.modelId needed ?

Reply:
If we have deviceInfo we should have device as well, it's fine